### DISCUSSION TITLE
unpkg down

### DISCUSSION THREAD

**Discussion by vdvcoder** [alpinejs/alpine#3236] [3. Feature ideas]
Hello,

Today unpkg.com is down and some cdn's don't work anymore.
Is it possible to add the cdn's to https://www.jsdelivr.com/ if possible.

Thanks in advance!

**Comment by martinpl** [comment]
For confirmation:
https://github.com/mjackson/unpkg/issues/343

**Comment by ekwoka** [comment]
It already is available on jsdelivr: https://cdn.jsdelivr.net/npm/alpinejs

and esm.sh : https://esm.sh/alpinejs

**Reply by scottzirkel** [reply]
The main code is, but none of the plugins are available.

**Reply by SimoTod** [reply]
Nah, they are all available. The compiled versions are all on npm and any CDN that supports npm can be used.

https://cdn.jsdelivr.net/npm/@alpinejs/intersect (You can get the correct names from [here](https://github.com/alpinejs/alpine/blob/main/scripts/release.js#L74-L96))

**Reply by scottzirkel** [reply]
Ah, I was using https://www.jsdelivr.com/unpkg to find them. Perfect, thanks!
